spec/factories.rb

```diff
@@ -50,6 +50,7 @@
     user
     board
     description { "" }
+    editor_mode { 'rtf' }
```

Comment:
Is this the right default for factories?

Reply:
I think so! I don't think it matters much either way, actually, since we usually set the content directly when we need it (and then this only affects the rendering of the writtens, not the models).